std/exception.d

```diff
@@ -1478,6 +1478,34 @@ private bool isUnionAliasedImpl(T)(size_t offset)
         static assert( isUnionAliased!(S.A5, 1)); //a5.b1;
 }
 
+package string errnoString(int errno)
```

Comment:
@FreeSlave Adding `nothrow @system` on the function (and keep the return type as `string`) should make this PR good to go.